client/src/styles/PortfolioFilters.scss

```
@@ -284,6 +284,12 @@
             @include for-phone-only {
               padding: 0;
 
+              .multiselect__menu,
+              .multiselect__control,
+              .multiselect__selected-value-control-container {
+                width: 75%;
+              }
+
```

there are some classnames that are redeclared after this block. consolidate like this?
`
```suggestion
          .multiselect__input-container,
          .multiselect__placeholder {
                font-size: 1.8rem;
                line-height: 100%;
                height: 5rem;
                padding-left: 1.2rem;
              }

              .multiselect__menu {
                font-size: 1.6rem;
              }
```

---

Thanks! just incorporated that